CMakeLists.txt

```diff
@@ -40,17 +50,18 @@ file(GLOB_RECURSE sources CONFIGURE_DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/source/
 # ---- Create library ----
 
 # Note: for header-only libraries change all PUBLIC flags to INTERFACE and create an interface
-# target: add_library(Greeter INTERFACE) set_target_properties(Greeter PROPERTIES
-# INTERFACE_COMPILE_FEATURES cxx_std_17)
-
+# target! EITHER: add_library(Greeter INTERFACE) OR:
 add_library(Greeter ${headers} ${sources})
 
-set_target_properties(Greeter PROPERTIES CXX_STANDARD 17)
+# EITHER: set_target_properties(Greeter PROPERTIES CXX_STANDARD 17) # OR:
```

Comment:
IMO this comment can be removed.

Reply:
ok, but I prefer to prevent `set_target_properties`() in most cases